pages/fiscal-sponsorship/index.js

```diff
@@ -96,11 +101,29 @@ function OpenSourceAlert() {
           flexDirection: ['column', 'row']
         }}
       >
+        <Box
+          sx={{
+            position: 'absolute',
+            top: 15,
+            right: -30,
+            bg: 'red',
+            color: 'white',
+            transform: 'rotate(45deg)',
+            width: 120,
+            textAlign: 'center',
+            py: 1,
+            fontSize: 1,
+            zIndex: 40,
+            fontWeight: 'bold',
+            boxShadow: '0 2px 4px rgba(0,0,0,0.2)'
+          }}
+        >
+          New!
+        </Box>
         <span style={{ fontSize: 20 }}>
-          <strong style={{ fontSize: 23 }}>HCB is open source! </strong>
+          <strong style={{ fontSize: 23 }}>HCB Mobile is here!</strong>
           <br />
-          Join us in building the infrastructure powering student-led
-          organizations
+          View and manage your HCB organizations on the go. Issue cards, view transactions, and more!
```

Comment:
"View" twice here felt repetitive.
```suggestion
          Manage your HCB organizations on the go. Issue cards, view transactions, and more!
```